templates/flat-square-template.svg

```diff
@@ -4,7 +4,12 @@
     <rect x="{{=it.widths[0]}}" width="{{=it.widths[1]}}" height="20" fill="{{=it.escapeXml(it.colorB||"#4c1")}}"/>
   </g>
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
-    <text x="{{=it.widths[0]/2+1}}" y="14">{{=it.escapeXml(it.text[0])}}</text>
+    {{?it.logo}}
+      <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
```

Comment:
Your use of `xlink` warrants the addition of a `xmlns:xlink="http://www.w3.org/1999/xlink"` in the SVG tag.